aut_docs/Installation_Guide_WindowsOS.md

```diff
@@ -173,17 +170,15 @@
     ```
     
 - Check that the project is running correctly by browsing to
-    ```
-       http://127.0.0.1:8000
-    ```
-  Note: The default web browser is Firefox.
+  [http://127.0.0.1:8000](http://127.0.0.1:8000)
+
+  **Note:** The default web browser is Firefox
     
 ### Steps to run tests
 
 - Automated tests in VMS require you to setup geckodriver in your path. For that, download geckodriver v0.20.0
-    ```
-       [Geckodriver v0.20.0](https://github.com/mozilla/geckodriver/releases)
-    ```
+  [Geckodriver v0.20.0](https://github.com/mozilla/geckodriver/releases)
```

Comment:
This one too! Remove the repeating word Geckodriver and hyperlink the geckodriver word after 'download' directly